doc/ref_model/chapters/chapter08-annex.md

```diff
@@ -6,232 +6,133 @@
 ## Table of Contents
 * [8.1 Introduction.](#8.1)
 * [8.2 Generic Test Cases.](#8.2)
-* [8.3 Architecture specfiic test cases (if needed).](#8.3)
+* [8.3 Architecture specific test cases (if needed).](#8.3)
   * [8.3.1 OpenStack (OS).](#8.3.1)
   * [8.3.2 Kubernetes (K8s).](#8.3.2)
 
 <a name="8.1"></a>
 ## 8.1 Introduction
 
-This Chapter 8 Annex contains standard test cases to be executed furing NFVI valications to Ensure a Reference Implementation of CNTT Reference Model and CNTT Reference Architecture meets industry driven quality assurance standards for compliance and verification.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), in partnership with the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA/RI verifications.
+This Chapter 8 Annex contains standard test cases to be executed during NFVI validations to ensure a Reference Implementation of CNTT Reference Model Architecture complies with industry-driven standards.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), in partnership with the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA/RI verifications.
```

Comment:
- I think these are not standard test cases, but only test cases
- There is a CNTT Reference Model and there are CNTT Reference Architetctures, but there is no CNTT Reference Model Architecture. These tests are either testing the CNTT Reference Model or one or more of the CNTT Reference Architetures.

Reply:
Updated language.